oneview_redfish_toolkit/api/network_device_function.py

```diff
@@ -59,8 +59,11 @@ def __init__(self, device_id, device_function_id, server_hardware):
             virtual_port = self.get_resource_by_id(
                 port["virtualPorts"], "portNumber", virtual_port_number)
         except Exception:
-            raise OneViewRedfishResourceNotFoundError(
```

Comment:
In this case, I believe the `OneViewRedfishResourceNotFoundError` is a good exception to raise.